test/Libraries/Microsoft.Extensions.AI.Integration.Tests/ChatClientIntegrationTests.cs

```diff
@@ -18,61 +18,41 @@
 using Microsoft.Extensions.Caching.Memory;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Logging.Testing;
-using Microsoft.TestUtilities;
 using OpenTelemetry.Trace;
+using Microsoft.DotNet.XUnitExtensions;
```

Comment:
Please revert all your changes to this file except of this `using` change. It's some major hallucination.